WooCommerce/Classes/POS/Presentation/POSFloatingControlView.swift

```diff
@@ -31,7 +31,7 @@ struct POSFloatingControlView: View {
                 }
             } label: {
                 Image(systemName: "ellipsis")
-                    .font(.posBodyEmphasized)
+                    .font(.posBodyEmphasized, maximumContentSizeCategory: .accessibilityLarge)
                     .foregroundStyle(fontColor)
                     .frame(width: Constants.size, height: Constants.size)
```

Comment:
Perhaps we should scale the frame, rather than limit the size?

Reply:
I'll check. What I found with `SF Symbols` is that `frame` doesn't work well, unless we need to apply some extra modifiers to enforce it.

Reply:
Yeah. I think the frame here is an extra – even though it's a modifier on the image, it defines the size of the button, rather than the symbol, which is controlled by the font.

Reply:
In the end, changing whole elpsis view size would mean that we would also need to change the size of the whole floating button. For now, we decided against it p91TBi-bYt-p2#comment-12893.